tests/test_vectors.json

```diff
@@ -153310,5 +153310,203 @@
                 "exported_value": "69eb994ac941859abf71c86623abd13040fea633a4da115195bb3eb59417f8d0"
             }
         ]
-    }
+    },
+    {
+        "mode": 0,
```

Comment:
I've added the `base` mode test vectors here. 

They came from [this doc](https://www.ietf.org/archive/id/draft-wahby-cfrg-hpke-kem-secp256k1-01.html#name-test-vectors). 

Given my lack of experience with vectors, I wanted to point out a few things:

1. I had to add a few fields to the vectors from that doc to make them match the format and work here.
    1. The `enc` field is the same as the `pkEm` field. 
    2. The `key_schedule_context` and `secret` fields are blank.
    3. The `encryption` and `exports` fields are also blank.